config/jobs/kubernetes/release/release-config.yaml

```diff
@@ -89,7 +89,7 @@ presubmits:
       testgrid-alert-email: release-managers+alerts@kubernetes.io
       testgrid-num-columns-recent: '30'
   - name: pull-release-image-kube-cross
-    optional: true
+    optional: false
```

Comment:
```suggestion
```

You can drop all of these lines, since they default to `false`.

Reply:
👍